### PR TITLE
Port data binding

### PR DESCRIPTION
**Pull request by yanana**

### CODE REVIEW

File: src/main/scala/org/aerospiker/json/package.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.aerospiker
+
+import org.aerospiker.{ Client => AClient }
+import org.aerospiker.policy.ClientPolicy
+
+package object json {
```
Comment (tkrs) [inline src/main/scala/org/aerospiker/json/package.scala:6]: `client` better than `json` ?

File: src/main/scala/org/aerospiker/json/Database.scala [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package org.aerospiker.json
+
+import cats.data.{ Ior, NonEmptyList => NEL, Xor }
+import org.aerospiker.Key
+
+import scalaz.concurrent.{ Future, Task }
```
Comment (tkrs) [inline src/main/scala/org/aerospiker/json/Database.scala:6]: `Future` is not use

Comment (yanana) [inline src/main/scala/org/aerospiker/json/Database.scala:6]: Exactly! :octopus:

File: src/main/scala/org/aerospiker/json/package.scala (hunk 1 above)
Comment (yanana) [inline src/main/scala/org/aerospiker/json/package.scala:6]: I don't want any more _client_! How about _codec_?

**Comment by tkrs** [comment]
LGTM

**Pull request merged**